site/src/css/utilities/patches.css

```
@@ -0,0 +1,6 @@
+/* Temporary (hopefully!) patches and work-arounds to override/undo issues from 3rd party CSS */
+
+/* Prevent Mosaic theme from inverting all images */
+[data-mode=dark] main img:not(.no-filter) {
```

Is this specific enough? In my change, I use this:

```suggestion
.salt-theme[data-mode=dark] main img:not(.no-filter) {
```

so no matter the CSS order, it will be more specific